src/components/sbb-divider/sbb-divider.stories.js

```diff
@@ -0,0 +1,121 @@
+import { h } from 'jsx-dom';
+import readme from './readme.md';
+import { SbbColorBlackDefault, SbbColorWhiteDefault } from '@sbb-esta/lyne-design-tokens';
+
+const documentationPlatformContainerStyle = (context) => {
+  if (context.args.negative) {
+    return {};
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const wrapperStyle = (context) => {
+  if (context.args.negative) {
+    return `background-color: ${SbbColorBlackDefault};`;
+  }
+
+  return `background-color: ${SbbColorWhiteDefault};`;
+};
+
+const Template = (args) => (
+  <div style="height: 340px; padding: 20px;">
+    <sbb-divider {...args} />
+  </div>
+);
+
+const orientation = {
+  control: {
+    type: 'select',
+  },
+  options: ['horizontal', 'vertical'],
+};
+
+const negative = {
+  control: {
+    type: 'boolean',
+  },
+  table: {
+    category: 'Appearance',
+  },
+};
+
+const defaultArgTypes = {
+  orientation,
+  negative,
+};
+
+const defaultArgs = {
+  orientation: orientation.options[0],
+  negative: false,
+};
+
+export const dividerDefault = Template.bind({});
+
+dividerDefault.args = JSON.parse(JSON.stringify(defaultArgs));
+dividerDefault.argTypes = defaultArgTypes;
+dividerDefault.documentation = {
+  container: {
+    styles: (context) => documentationPlatformContainerStyle(context),
+  },
+  title: 'Divider Default',
+};
+
+export const dividerNegative = Template.bind({});
+
+dividerNegative.argTypes = defaultArgTypes;
+dividerNegative.args = {
+  ...defaultArgs,
+  negative: true,
+};
+dividerNegative.documentation = {
+  container: {
+    styles: (context) => documentationPlatformContainerStyle(context),
+  },
+  title: 'Divider Negative',
+};
+
+export const dividerHorizontal = Template.bind({});
```

Comment:
Divider default and divider horizontal seem to be the same story?

Reply:
Removed 👍 The difference was default was called with default args while horizontal was called with horizontal args but yeah it's the same story.